### PR TITLE
Update booking status

### PR DESCRIPTION
**Pull request by ellie-xyb**
With this pr, the dashboard can call it mostly done.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -8,5 +8,5 @@
     resources :pet_schedules, only: [:new, :create]
   end
   get "dashboard", to: 'dashboards#display'
-  resources :bookings, only: [:update, :show]
+  resources :bookings, only: [:update, :show, :update]
```
Comment (polanpio) [inline config/routes.rb]: we have update here twice

**Comment by polanpio** [comment]
I will work on styles from now on.

File: config/routes.rb (hunk 1 above)
Comment (ellie-xyb) [reply config/routes.rb]: I will fix it ASAP.

**Comment by ellie-xyb** [comment]
Fixed duplicate routes. May I merge this pr first?

**Pull request merged**